examples/metrics/src/main/java/io/opentelemetry/example/metrics/DoubleCounterExample.java

```diff
@@ -0,0 +1,53 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.metrics.DoubleCounter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.Tracer;
+
+
+/**
+ * Example of using {@link DoubleCounter} to measure execution time of method.
+ */
+public class DoubleCounterExample {
+
+  private static final Tracer tracer =
+      OpenTelemetry.getTracer("io.opentelemetry.example.metrics");
+
+  public static void main(String[] args) {
+    Span span = tracer.spanBuilder("superLongMethod")
+        .setSpanKind(Kind.INTERNAL)
+        .startSpan();
+    Meter sampleMeter = OpenTelemetry.getMeterProvider()
+        .get("io.opentelemetry.example.metrics", "0.5");
+    DoubleCounter timeCounter = sampleMeter.doubleCounterBuilder("some_method_time_usage")
+        .setDescription("should measure some method execution time")
+        .setUnit("second")
```

Comment:
Is a counter really the right instrument type to use here?
I thought this was only used for things that you actually count, like, the number of requests, errors or total bytes written or something like this. In particular, things where one would be interested in the total sum only and not distinct values or their min/max. Of course one could argue that measuring time entails counting seconds but I thought the ValueRecorder would be the correct instrument to record such distinct measurements.

The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#valuerecorder) even mentions:

>Example uses for ValueRecorder that are non-additive:
>- capture any kind of timing information

Also the example in the Javadoc even demonstrates how it's used to report timings:
https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/metrics/DoubleValueRecorder.java

Reply:
Do we know what a DoubleCounter would be used to count? Seems like an odd thing to count fractional stuff.

Reply:
I wonder, if we don't have a good use-case for it, if we should consider removing it from the API.

Reply:
@jkwatson Fair point. I think it makes sense to consider it but it might only be us not being able to come up with a good example. The example in DoubleCounter's Javadoc is slightly contrived but I could imagine that there are reasonable use cases for it out in the wild. Apart from that, removing it might be surprising to users for the lack of symmetry in the API then.